templates/default/vcs_ssh/sshd_config.erb

```diff
@@ -0,0 +1,12 @@
+AuthorizedKeysCommand /usr/libexec/ssh-phabricator-hook
+AuthorizedKeysCommandUser <%= node['phabricator']['vcs_ssh']['user'] %>
+
+Port <%= node['phabricator']['vcs_ssh']['port'] %>
+Protocol 2
+PermitRootLogin no
+AllowAgentForwarding no
+AllowTcpForwarding no
+PrintMotd no
```

Comment:
Would be nice to have `X11Forwarding no`, even though it is the default, to be explicit about it.